Client/src/components/MovieTile/movieTile.tsx

```diff
@@ -30,6 +30,19 @@ const MovieTile = (props:MovieTileProps) => {
         setIsMouseOverTile(prev => prev = false)
     }
 
+    const showMoreButton = () => {
+        if("duration" in props){
```

Comment:
Można to zmienić jako
`const path = "duration" in props ? '/movie/m/${props.id}':  "seasons" in props ? '/movie/s/${props.id}'

return (
        <div className="icon iconButton moreButton" onClick={() => navigate(path)} />
    );
`

Reply:
Zrobione